mediator/src/controllers/openmrs.ts

```diff
@@ -0,0 +1,15 @@
+import { logger } from '../../logger';
+import { syncPatients, syncEncounters } from '../utils/openmrs_sync'
+
+export async function sync() {
+  try {
+    const startTime = new Date();
+    startTime.setHours(startTime.getHours() - 1);
```

Comment:
Why set start time to an hour ago?

Reply:
One hour was supposed to be form how long ago the sync should look for new resources. But it should be configureable, I have added an environment vairable SYNC_PERIOD for this instead.